### DISCUSSION TITLE
Always being redirected to http://mydomain:2000/?origin=host&app=appname&path=%2F

### DISCUSSION THREAD

**Discussion by phelipebf** [Radiokot/umbrel-cloudflared#14] [General]
My hostnames always redirects to http://mydomain:2000/?origin=host&app=myspeed&path=%2F and never load page.

![image](https://github.com/user-attachments/assets/fe4ee37d-3d58-47ed-b134-c2c3ab7ab3ce)

![image](https://github.com/user-attachments/assets/180b0890-2cdc-4a1f-b165-a3d2dc69e0bb)

**Comment by Radiokot** [comment]
Please, follow the guide on exposing protected apps:
https://github.com/Radiokot/umbrel-cloudflared/wiki/How-to-set-up-Cloudflare-Tunnel-on-your-Umbrel#advanced-route-to-a-protected-app

**Reply by phelipebf** [reply]
Solved!

Thank you!
